src/meson.build

```diff
@@ -35,21 +18,14 @@ clippy_sources += gnome.compile_resources(
     c_name: 'clippy'
 )
 
-clippy_lib = shared_library('clippy-' + api_version,
+gtk_modules_path = join_paths(gtk_name,
+                              gtk_dep.get_pkgconfig_variable('gtk_binary_version'),
+                              'modules')
+
+clippy_lib = shared_library(
```

Comment:
I think this should be `shared_module`, otherwise it might get versioned.

Reply:
I will check this out.